.gitignore.orig

```diff
@@ -0,0 +1,47 @@
+# Ignore possible build & documentation
+docs/_build/
+build/
+
+# Ignore temp files
+*.pyc
+*.save
+*.bak
+.*.swp
+*~
+*.o
+*.mod
+*.so
+
+# Ignore data &binary files
+*.h5
+*.ace
+
+# Ignore specific files
+matplotlibrc
+.ipynb_checkpoints
+
+# Ignore rope files introduced by python-mode
+.ropeproject
+<<<<<<< HEAD
```

Comment:
this line, starting with <<<< , remains from an incomplete merge. That is, git inserts this text to differentiate between the conflicting changes in two revisions. This line (25) and all those below it should be deleted.